README.md

```diff
@@ -1,14 +1,46 @@
-# README
+# sceptre-request-resolver
 
-Add your resolver readme here. Remember to include the following:
+A Sceptre resolver to make requests from REST API endpoints.
 
-- Tell people how to install it (e.g. pip install ...).
-- Be clear about the purpose of the resolver, its capabilities and limitations.
-- Tell people how to use it.
-- Give examples of the resolver in use.
+## Motivation
 
-Read our wiki to learn how to use this repo:
-https://github.com/Sceptre/project/wiki/Sceptre-Resolver-Template
+There are some pretty useful REST API endpoints on the internet.  The endpoints
+can return lots of different types of data, typically in JSON format.
+This simple resolver can retrieve that data and pass it to Sceptre parameters
+or scepter_user_data parameters.
 
-If you have any questions or encounter an issue
-[please open an issue](https://github.com/Sceptre/project/issues/new)
+## Installation
+
+To install directly from PyPI
+```shell
+pip install sceptre-request-resolver
+```
+
+To install from this git repo
+```shell
+pip install git+https://github.com/Sceptre/sceptre-request-resolver.git
+```
+
+## Usage/Examples
+
+```yaml
+parameters|sceptre_user_data:
+  <name>: !request <API ENDPOINT>
+```
+
+## Example
+
+Simple request:
+```yaml
+parameters:
+  wisdom: !request 'https://ron-swanson-quotes.herokuapp.com/v2/quotes'
+```
+__Note__: request returns a string.
+
+
+Request data and convert the response string to a JSON object:
+```yaml
+sceptre_user_data:
+   hounds: !from_json_string {response}
```

Comment:
This resolver doesn't exist yet. Are you planning to introduce that one separately and release this resolver after it gets released?

Reply:
hmm, maybe i should just remove this from the docs so that we can release this resolver now.  then add the doc back in after @dboitnot implements and releases `!from_json_string`?

Reply:
I'd prefer that route. Maybe include a comment in the docs that says the resolver always returns a string, so any returned json will need later deserialization if the user desires it deserialized.